src/level-ext.adoc

```diff
@@ -0,0 +1,156 @@
+|Bits[4:3]| R | W | C | LM | EL | SL  | X | ASR | Mode^1^ |
+| 0-1   | ✔ | ✔ | ✔ | ✔  | ✔  | ∞   | ✔ |  ✔  | Mode^1^  | Execute + ASR (see <<infinite-cap>>)
+| 2-3   | ✔ |   | ✔ | ✔  | ✔  | ∞   |   |     | Mode^1^  | Execute + Data & Cap RO
+| 4-5   | ✔ | ✔ | ✔ | ✔  | ✔  | ∞   | ✔ |     | Mode^1^  | Execute + Data & Cap RW
+| 6-7   | ✔ | ✔ |   |    |    | N/A |   |     | Mode^1^  | Execute + Data RW
+11+| *Quadrant 2: Restricted capability data read/write*
+11+| bit[2] = write, bit[1:0] = store level. R and C implicitly granted, LM dependent on W permission.
+|Bits[4:3]| R | W | C | LM | EL | SL    | X | ASR | Mode^1^ |
+| 0-2   10+| reserved
+| 3       | ✔ |   | ✔ |    |    | N/A   |   |     | N/A | Data & Cap R0 (without <<lm_perm>>)
+| 4       | ✔ | ✔ | ✔ | ✔  |    | _(3)_ |   |     | N/A | Reserved for `LVLBITS=2`
+| 5       | ✔ | ✔ | ✔ | ✔  |    | _(2)_ |   |     | N/A | Reserved for `LVLBITS=2`
+| 6       | ✔ | ✔ | ✔ | ✔  |    | 1     |   |     | N/A | Data & Cap RW (with store _local_, no <<el_perm>>)
+| 7       | ✔ | ✔ | ✔ | ✔  |    | 0     |   |     | N/A | Data & Cap RW (no store _local_, no <<el_perm>>)
+11+| *Quadrant 3: Capability data read/write*
+11+| bit[2] = write, bit[1:0] = store level. R and C implicitly granted.
+11+| _Reserved bits for future extensions must be 1 so they are implicitly granted_
+|Bits[4:3]| R | W | C | LM | EL | SL    | X | ASR | Mode^1^ |
+| 0-2   10+| reserved
+| 3       | ✔ |   | ✔ | ✔  | ✔  | N/A   |   |     | N/A | Data & Cap R0
+| 4       | ✔ | ✔ | ✔ | ✔  | ✔  | _(3)_ |   |     | N/A | Reserved for `LVLBITS=2`
+| 5       | ✔ | ✔ | ✔ | ✔  | ✔  | _(2)_ |   |     | N/A | Reserved for `LVLBITS=2`
+| 6       | ✔ | ✔ | ✔ | ✔  | ✔  | 1     |   |     | N/A | Data & Cap RW (with store _local_)
+| 7       | ✔ | ✔ | ✔ | ✔  | ✔  | 0     |   |     | N/A | Data & Cap RW (no store _local_)
+|==============================================================================
+
+[#section_cap_level_change]
+=== Changing capability levels and permissions
+While capability levels are conceptually a label on the capability rather than a permission granted by the capability, they are adjusted using the <<ACPERM>> instruction.
+This avoids the need for a dedicated instruction and allows reducing the level and removing <<el_perm>> in a single instruction.
+
+<<<
+
+[#section_cap_level_summary]
+=== Capability level summary table
+
+.{cheri_levels_ext_name} `LVLBITS=1` summary table for stored capabilities
+[#cap_level_store_summary,width="100%",options=header,align="center",cols="1,1,1,1,5"]
+|==============================================================================
+3+|Auth cap field | Data cap field | Notes
+|*W*|*C*|*SL*|*CL*|
+|1  | 1 | 1  | X  | Store data capability unmodified
+|1  | 1 | 0  | 1  | Store data capability unmodified
+|1  | 1 | 0  | 0  | Store data capability with tag cleared
+|==============================================================================
```

Comment:
I would make this table like this instead:

|SL|CL| ~SL | Tag|
|-|-|-|-|
| 0 | 0 | 1 | 0 |
| 0 | 1 | 1 | Unchanged
| 1 | 0 | 0 | Unchanged
| 1 | 1 | 0 | Unchanged

Reply:
I've updated the table now and kept the ~SL as part of a comment instead which is easier to read IMO.